src/pages/[platform]/build-a-backend/data/set-up-data/index.mdx

```diff
@@ -80,26 +80,33 @@ To deploy these resources to your cloud sandbox, run the following CLI command i
 
 <InlineFilter filters={["react", "angular", "javascript", "vue", "nextjs", "react-native"]}>
 
-```bash
+```bash title="Terminal" showLineNumbers={false}
 npx amplify sandbox
 ```
 
 </InlineFilter>
 
 <InlineFilter filters={["android"]}>
 
-```bash
-npx amplify sandbox --config-format json-mobile --config-out-dir <path_to_src/main/res/raw_folder>
+```bash title="Terminal" showLineNumbers={false}
+npx amplify sandbox --config-format json-mobile --config-out-dir <path_to_android_project>/src/main/res/raw/
 ```
 
 </InlineFilter>
 
 <InlineFilter filters={["swift"]}>
 
-```bash
+```bash title="Terminal" showLineNumbers={false}
 npx amplify sandbox --config-format json-mobile --config-out-dir <path_to_swift_project>
```

Comment:
This does not work unfortunately. Once the Json is generated, it needs to be moved manually.

Reply:
I can confirm this worked "on my machine" (tm) 😅